### ISSUE TITLE
Issue with Navbar

### ISSUE THREAD

**Issue by Vaibhav-afk** [Namanl2001/MERN-Gurujii-dev#131]
**Description of issue**
I don't see any change or some actions (like redirecting to another page) when I click on `Link` or any option from dropdown menu.   

**Expected behavior**
Something must happen, when we click those options.

**Comment by Vaibhav-afk** [comment]
Can you just confirm me whether I have to remove those options or you have added those for some reason.

**Comment by sagar-barapatre** [comment]
We have to add something. Yet not decided but surely in future the links will be updated.

**Comment by Vaibhav-afk** [comment]
So I think it's better to remove or comment out the code as it would be a bad user experience because user is not aware of this.

**Comment by Namanl2001** [comment]
Basically, I've just added them to fill some space on the navbar 😅 otherwise it looks totally empty. Also clicking on them have no actions. So should we remove them completely? WDYT?

**Comment by Vaibhav-afk** [comment]
@Namanl2001 yeah, I understand but suppose you visit some site and you clicked some button you will expect something from this action but if it  doesn't do anything, you will say the site has bug. I hope you are getting what I am trying to convey, its all about user experience.
